package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "react-calculator",
+  "version": "1.0.0",
+  "description": "In the initial commit to this project you have been provided with only an ESLint config for a React project. Your task is to create an interactive React Calculator page that responds to user input.",
+  "main": ".eslintrc.js",
+  "scripts": {
+    "dev": "./node_modules/.bin/webpack --progress --inline --config webpack.dev.js",
+    "lint": "./node_modules/.bin/eslint --format codeframe --ext jsx ."
```

Comment:
You added this command but apparently didn't use it

![image](https://user-images.githubusercontent.com/5775716/83040837-b92d6380-a00d-11ea-83b4-fb4f633509d3.png)